config/environments/development.rb

```diff
@@ -34,4 +34,6 @@
   config.assets.quiet = true
 
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  config.stripe.secret_key = ENV['STRIPE_SECRET_KEY']
```

Comment:
To be able test stripe, we need a secret api key to test it. In this file we set it for the development environment.